Source/Core/DolphinQt2/Translator.h

```diff
@@ -0,0 +1,262 @@
+// Copyright 2015 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QString>
+#include <QTranslator>
+
+class Translator final : public QTranslator
+{
+public:
+	QString translate(const char* context, const char* source_text, const char* disambiguation = 0, int n = -1) const override;
+	static void SetLanguage(int languageID);
+	static void Init();
+	static const char* GetLocale(int languageID);
```

Comment:
I would make it private for now. If we need it later we can make it public again later.